src/testing/_asset.ts

```diff
@@ -1,6 +1,4 @@
-import { Account, Algodv2, Kmd, makeAssetCreateTxnWithSuggestedParamsFromObject } from 'algosdk'
-import { algos, microAlgos } from '../amount'
-import { ensureFunded } from '../transfer'
+import { Account, Algodv2, makeAssetCreateTxnWithSuggestedParamsFromObject } from 'algosdk'
```

Comment:
Renamed this file to `_asset.ts` to reflect the fact that we don't want to export these methods outside of algokit